flagsmith-engine/evaluationContext/models.ts

```diff
@@ -0,0 +1,42 @@
+import type {
+    EnvironmentContext,
+    IdentityContext,
+    SegmentContext,
+    SegmentRule,
+    SegmentCondition,
+    InSegmentCondition,
+    FeatureContext,
+    FeatureValue as ContextFeatureValue,
+    Traits,
+    Features,
+    Segments
+} from './evaluationContext.types.ts';
+
+export type EnvironmentKey = EnvironmentContext['key'];
```

Comment:
Added this manually to avoid relying on auto-generated redundant name keys:
```
export type Key1 = string;
/**
 * Key used for % split segmentation.
 */
export type Key2 = string;
/**
```